public/components/overview/mitre_attack_intelligence/module_mitre_attack_intelligence_all_resources.tsx

```diff
@@ -11,21 +11,53 @@
  * Find more information about this on the LICENSE file.
  */
 
-import React from 'react';
+import React, {useCallback, useState} from 'react';
 import { 
   EuiTitle,
   EuiSpacer
 } from '@elastic/eui';
 import { withGuard } from'../../../components/common/hocs';
 import { ModuleMitreAttackIntelligenceAllResourcesWelcome } from './module_mitre_attack_intelligence_all_resources_welcome';
 import { ModuleMitreAttackIntelligenceAllResourcesSearchResults } from './module_mitre_attack_intelligence_all_resources_search_results';
+import { ModuleMitreAttackIntelligenceFlyout } from './module_mitre_attack_intelligence_resource_flyout';
 
 export const ModuleMitreAttackIntelligenceAllResources = withGuard(({didSearch}) => !didSearch, ModuleMitreAttackIntelligenceAllResourcesWelcome)(({ results, loading }) => {
+  const [isDetailsOpen, setIsDetailsOpen] = useState(false);
```

Comment:
Should you use types for useStates like:
`  const [isDetailsOpen, setIsDetailsOpen] = useState<boolean>(false);
`
Same for the item if it possible create an `interface WzItem { ... }`  and assigne that.